src/Stripe.net/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -123,10 +128,15 @@ public virtual async Task<StripeInvoice> UpdateAsync(string invoiceId, StripeInv
         }
 
         public virtual async Task<StripeInvoice> PayAsync(string invoiceId, StripeRequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await this.PayAsync(invoiceId, null, requestOptions, cancellationToken);
```

Comment:
Actually, I did some research and the right approach here is not declare this method as `async` and not use `await`. Can you update accordingly? Sorry for the extra work :/

Reply:
Hmmm, are you sure that method will still appear as Async though and work as expected?

Reply:
On third thought, it would work, but it would be a breaking change :/

Let's just get rid of the overloads and release this as a major version?

Reply:
Done!